README.md

```diff
@@ -47,6 +47,7 @@ It will download the last version of every file present on Wayback Machine to `.
 	    -p, --maximum-snapshot NUMBER    Maximum snapshot pages to consider (Default is 100)
 					     Count an average of 150,000 snapshots per page
 	    -l, --list                       Only list file urls in a JSON format with the archived timestamps, won't download anything
+	    -u, --user-agent STRING          UserAgent for connection (Default is WayBack Machine Downloader)
```

Comment:
Dont forget to update the readme and in-file commands is any with the default UA

Reply:
oh, fixed